SourceryTests/Stub/JavaScriptTemplates/Includes.js

```diff
@@ -0,0 +1 @@
+<%- include('Equality.js') %>-%><%- include('Other.js') %>-%>
```

Comment:
Yes, that's a newline slurp tag, which is needed to match the expected result (otherwise it appends a newline)